packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m

```diff
@@ -114,6 +114,10 @@ - (instancetype)initWithURL:(NSURL *)url
                 httpHeaders:(nonnull NSDictionary<NSString *, NSString *> *)headers
                   avFactory:(id<FVPAVFactory>)avFactory
                   registrar:(NSObject<FlutterPluginRegistrar> *)registrar;
+
+// Tells the player to run its frame updater until it receives a frame, regardless of the
+// play/pause state.
+- (void)expectFrame;
```

Comment:
nit: maybe `startDisplayLinkIfNeeded`?

Reply:
That would not be an accurate name; `startDisplayLinkIfNeeded` would just call `displayLink.running = true;`. If this were named by implementation, it would have to be called something like `startDisplayLinkIfNeededAndPreventStoppingUntilNextFrameAvailable`.

This is named by its conceptual purpose for callers, which is to instruct the player to expect that a frame will become available (and to handle state accordingly to make sure it's rendered), rather than its implementation.